saleor/payment/gateways/braintree/__init__.py

```diff
@@ -234,3 +276,27 @@ def process_payment(
 ) -> GatewayResponse:
     auth_resp = authorize(payment_information, config)
     return auth_resp
+
+
+def list_client_sources(
+    config: GatewayConfig, customer_id: str
```

Comment:
You should keep consistency between functions' params. If you introduce new dataclass then you should use it to pass `customer_id`, instead of direct passing `customer_id`. This will help us with keeping all generic method in similar design.

Reply:
There is no new dataclass here, GatewayConfig is not a place for customer_id.
Id for customer is precisely what every gateway will need to fetch stored sources, not sure what is more generic than that?